Modules/constants.c

```diff
@@ -48,38 +49,81 @@ LDAPerr(int errnum)
 
 /* Convert an LDAP error into an informative python exception */
 PyObject *
-LDAPerror(LDAP *l, char *msg)
+LDAPraise_for_message(LDAP *l, char *msg, LDAPMessage *m)
```

Comment:
The `msg` argument is unused, can it be removed?

Reply:
Thanks for spotting it. That wasn't intentional and should be fixed now.

Reply:
Ah, it was never used and https://github.com/python-ldap/python-ldap/blob/master/Lib/ldap/ldapobject.py#L325 makes it impossible to actually propagate it without breaking/changing existing tests.

It feels quite wrong to it that way, but I guess it's unavoidable for now.